packages/server/src/handlers/cloud.ts

```diff
@@ -0,0 +1,29 @@
+import type { IncomingMessage, ServerResponse } from "http";
+import { LLamaCloudFileService } from "llamaindex";
+import { sendJSONResponse } from "../utils/request";
+
+export const getLlamaCloudConfig = async (
+  req: IncomingMessage,
+  res: ServerResponse,
+) => {
+  if (!process.env.LLAMA_CLOUD_API_KEY) {
```

Comment:
also use getEnv here in this file
```suggestion
  if (!getEnv("LLAMA_CLOUD_API_KEY")) {
```